app/helpers/participant_helper.rb

```diff
@@ -26,7 +26,7 @@ def us_states
   def registry_details_formatter(participant)
     [
       "<a class='details participant-details ml10' href='javascript:void(0)' title='Details' participant_id='#{participant.id}'>",
-      "<i class='glyphicon glyphicon-sunglasses'></i>",
+      "<i class='fa fa-info'></i>",
```

Comment:
Minor detail: is it possible to replace this line with the `icon` helper to be more consistent with the other places we use fontawesome icons?